apps/extension/src/pages/settings/layout.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link, Outlet } from "@tanstack/react-router";
+
+export const SettingsLayout = () => {
+  return (
+    <div className="w-full mx-2">
+      <h2 className="text-xl font-bold text-gray-600 pt-3 pb-1 px-3">
+        Settings
+      </h2>
+
+      <div
+        className="flex text-gray-600 gap-2"
+        style={{ boxShadow: "inset 0 -1px 0px 0px #d6d6d6" }}
```

Comment:
I took it as-is from the frontend repo. It would probably just be arbitrary values anyway and I don't really like this: `shadow-[0_-1px_0_-15px_#d6d6d6]`

Reply:
done